### PR TITLE
[HER-82] Create camera repository

### PR DESCRIPTION
**Pull request by Urzica-Platon-Casian**
## Description
It will live inside data layer
It will be used for fetching and creating cameras by business command
Configure camera factory
Create insert method

## Related
[HER-82](https://nerds-interns.atlassian.net/jira/software/projects/HER/boards/1?issueParent=10022&selectedIssue=HER-82)

### CODE REVIEW

**Review by danidoff — DISMISSED** [review]

File: src/Data/Camera/CameraFactory.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        };
+
+        public static ECamera MakeEntity(MRegisterCamera model) => new ECamera()
+        {
+            Id = model.Id,
+            Latitude = model.Latitude,
+            Longitude = model.Longitude,
+        };
+    }
+}
```
Comment (RaduBuciuceanu) [inline src/Data/Camera/CameraFactory.cs:22]: Should we also create unit tests for this factory?

**Review by RaduBuciuceanu — CHANGES_REQUESTED** [review]

File: src/Data/Camera/CameraFactory.cs (hunk 1 above)
Comment (Urzica-Platon-Casian) [reply src/Data/Camera/CameraFactory.cs:22]: Done

File: test/Data.UnitTests/Camera/CameraFactoryTest.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    using Business.Camera.Register.Models;
+    using Data.Camera;
+    using NUnit.Framework;
+    using Shouldly;
+
+    public class CameraFactoryTest
+    {
+        private MCamera _model;
+        private ECamera _entity;
+        private MRegisterCamera _modelCreate;
```
Comment (RaduBuciuceanu) [inline test/Data.UnitTests/Camera/CameraFactoryTest.cs:13]: Would the name `_input` be more relevant here?

**Review by RaduBuciuceanu — CHANGES_REQUESTED** [review]

File: test/Data.UnitTests/Camera/CameraFactoryTest.cs (hunk 2 above)
Comment (Urzica-Platon-Casian) [reply test/Data.UnitTests/Camera/CameraFactoryTest.cs]: Done

**Review by RaduBuciuceanu — DISMISSED** [review]

**Review by RaduBuciuceanu — APPROVED** [review]

**Pull request merged**